src/_pytest/terminal.py

```diff
@@ -181,7 +184,8 @@ def getreportopt(config: Config) -> str:
             reportopts = "sxXwEf"
         elif char == "A":
             reportopts = "PpsxXwEf"
-            break
+        elif char == "N":
+            reportopts = ""
```

Comment:
If I'm reading this right, it means that `-rN` also implies `--disable-warnings`. Is that intended?

Reply:
@The-Compiler yes.  See https://github.com/pytest-dev/pytest/issues/5066.
I've amended/extended the tests a bit.